doc/dataset-conversions.rst

```diff
@@ -0,0 +1,125 @@
+********
```

Comment:
There are a couple of formatting errors in this text. The main one I see is that to mark titles in RST, you have to make the markers as long as the title e.g.
```
***************************
This is a really long title
***************************
```